i18n/en.json

```diff
@@ -361,7 +361,7 @@
   },
   {
     "id": "api.channel.remove_member.removed",
-    "translation": "%v was removed from the channel."
+    "translation": "%v was removed from the team."
```

Comment:
This message shouldn't be changed. We should have a new message for when the user is removed from the town square/team

Reply:
Ok I am gonna revert it but should I add the string "api.team.remove_member.removed" then ?